addon/components/sl-button.js

```diff
@@ -145,45 +156,57 @@ export default Ember.Component.extend( TooltipEnabled, {
         if ( label ) {
             return label;
         }
-    }.property( 'label', 'pending', 'pendingLabel' ),
+    }),
 
     /**
      * Converted size string to Bootstrap button class
      *
-     * @function sizeClass
+     * @function
      * @observes size
-     * @returns  {Ember.String} Defaults to undefined
+     * @returns {?String} Defaults to undefined
```

Comment:
?Type just denotes that the value is nullable, not what the default is necessarily.